index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FanExpo San Francisco 2023</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/69940923dd.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <header class="header-mobile">
+      <nav>
+        <div class="hamburger-menu">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+      </nav>
+    </header>
+    <header class="mobile-menu">
+      <div class="close-icon"><i class="fa-solid fa-xmark"></i></div>
+      <div href="./about.html" class="menu-links">About</div>
+      <div href=".main-program" class="menu-links">Program</div>
+      <div href="#join-link" class="menu-links">Join</div>
+      <div href="./about.html/#partners-mobile" class="menu-links">Sponsor</div>
+      <div class="menu-links">News</div>
+    </header>
```

Comment:
- [ ] From home page on the mobile version. When clicking on the hamburger, the mobile menu would appear. It is expected to redirect a user from home page to about page when `about` menu item is clicked. But the behaviour in your case is not. It remains on the home page despite clicking on the about menu item number of times. Kindly look into that and fix. 
[mobile-menu-home.webm](https://user-images.githubusercontent.com/26597432/212057560-ebf7fce2-0ef3-41cc-9d04-792172a880b8.webm)